tracer/src/Datadog.Trace/HttpOverStreams/DefaultJsonHeaderHelper.cs

```diff
@@ -0,0 +1,30 @@
+// <copyright file="DefaultJsonHeaderHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System.Linq;
+
+namespace Datadog.Trace.HttpOverStreams;
+
+internal class DefaultJsonHeaderHelper : HttpHeaderHelperBase
+{
+    private static string? _metadataHeaders = null;
+
+    protected override string MetadataHeaders
+    {
+        get
+        {
+            if (_metadataHeaders == null)
+            {
+                var headers = AgentHttpHeaderNames.DefaultHeaders.Select(kvp => $"{kvp.Key}: {kvp.Value}{DatadogHttpValues.CrLf}");
```

Comment:
Probably only want a subset of these headers?

Reply:
Headers changed here 360b832aff3dc02a2f871a46cb13bf52d25d75fe

Reply:
I think this is OK? 🤔